website/content/en/docs/cloud-providers/AWS/launch-templates.md

```diff
@@ -0,0 +1,236 @@
+---
+title: "Launch Templates and Custom Images"
+linkTitle: "Launch Templates"
+weight: 80
+---
+
+By default, Karpenter generates launch templates that use [EKS Optimized AMI](https://docs.aws.amazon.com/eks/latest/userguide/eks-optimized-ami.html) for nodes. Often, users need to customize the node image to integrate with existing infrastructure or meet compliance requirements. Karpenter supports custom node images through Launch Templates. If you need to customize the node, then you need a custom launch template. 
+
+## Introduction
+
+Karpenter follows existing AWS patterns for customizing the base image of
+instances. More specifically, Karpenter uses [EC2 launch templates](https://docs.aws.amazon.com/autoscaling/ec2/userguide/LaunchTemplates.html). Launch
+templates may specify many values. The pivotal value is the base image (AMI).
+Launch templates further specify many parameters related to networking,
+authorization, instance type, and more.  

```

Comment:
remove `instance type` because karpenter overrides the LT instance type, so it will never be used even if a customer specifies it in their custom LT. 

EDIT: I see you say never to specify the instance type in the LT Requirements section. I kind of still think it should be removed here to not confuse anyone. But I'll leave that up to you.